packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Legend/ReferenceLine.tsx

```diff
@@ -202,11 +202,11 @@ export const ReferenceLine: FC<Props> = ({
     }, [referenceLine]);
 
     const [value, setValue] = useState<string | undefined>(
-        referenceLine.data[markLineKey],
+        referenceLine.data[markLineKey] || referenceLine.data.type,
     );
 
     const [label, setLabel] = useState<string | undefined>(
-        referenceLine.data.label?.formatter,
+        referenceLine.data.name,
```

Comment:
Might need a back-compat check here

Reply:
yeah, this is risky, last time I changed something like this  I broke the ability to have multiple reference lines...

Reply:
I tested locally and it seems to be backwards compatible :+1: